.github/workflows/build-release.yml

```
@@ -135,328 +121,153 @@ jobs:
-
-        echo "$output_name<<$delimiter" | Out-File -FilePath $env:GITHUB_OUTPUT -Append
-        echo $changelog_content | Out-File -FilePath $env:GITHUB_OUTPUT -Append
-        echo $delimiter | Out-File -FilePath $env:GITHUB_OUTPUT -Append
+      env:
+        RELEASE_VERSION: ${{ needs.version.outputs.version }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: python tools/ci/prepare_release_assets.py --release-version "$env:RELEASE_VERSION"
 
     - name: Create Release
-      if: ${{ (github.event_name == 'workflow_dispatch' && inputs.create_release == true) || startsWith(github.ref, 'refs/tags/') }}
       uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)